Source/Core/InputCommon/ControllerEmu/ControlGroup/IMUAccelerometer.cpp

```diff
@@ -25,9 +25,16 @@ IMUAccelerometer::IMUAccelerometer(std::string name_, std::string ui_name_)
   AddInput(Translate, _trans("Backward"));
 }
 
+bool IMUAccelerometer::AreInputsBound() const
+{
+  return controls[0]->control_ref->BoundCount() && controls[1]->control_ref->BoundCount() &&
+         controls[2]->control_ref->BoundCount() && controls[3]->control_ref->BoundCount() &&
+         controls[4]->control_ref->BoundCount() && controls[5]->control_ref->BoundCount();
+}
+
 std::optional<IMUAccelerometer::StateData> IMUAccelerometer::GetState() const
 {
-  if (controls[0]->control_ref->BoundCount() == 0)
+  if (!AreInputsBound())
```

Comment:
FWIW I did the following in `IMUGyroscope`:

```
if (std::all_of(controls.begin(), controls.end(),
                  [](const auto& control) { return control->control_ref->BoundCount() == 0; }))
```

(and include `algorithm`)

Reply:
Ah, I see you changed `IMUGyropscope`.  I'll need to see why but I'd still suggest matching behavior if possible.

Reply:
True, I wrote this before someone implemented the std::all_of in IMUGyroscope and then didn't update it.